bemb/model/bemb.py

```diff
@@ -293,6 +300,18 @@ def __init__(self,
                 variation = coef_name.split('_')[-1]
                 mean = self.prior_mean[coef_name] if isinstance(
                     self.prior_mean, dict) else self.default_prior_mean
+
+                if isinstance(self.prior_variance, dict):
+                    # the user didn't specify prior variance for this coefficient.
+                    if coef_name not in self.prior_variance.keys():
+                        # the user may specify 'default' prior variance through the prior_variance dictionary.
+                        if 'default' in self.prior_variance.keys():
+                            warnings.warn(f"You provided a dictionary of prior variance, but coefficient {coef_name} is not a key in it. We fund a key 'default' in the dictionary, so we use the value of 'default' as the prior variance for coefficient {coef_name}.")
+                            self.prior_variance[coef_name] = self.prior_variance['default']
+                        else:
+                            warnings.warn(f"You provided a dictionary of prior variance, but coefficient {coef_name} is not a key in it. 'default' is not a key of prior variance dictionary, assuming unit variance for coefficient {coef_name}.")
```

Comment:
the second line here should be; 

Supply a value for 'default' in this dictionary to use that as default value; using variance=1.0 since this is not supplied.